emitter/bytes.go

```diff
@@ -0,0 +1,29 @@
+package emitter
+
+import (
+	"bytes"
+
+	"github.com/mccanne/zq/pkg/zsio"
+	"github.com/mccanne/zq/pkg/zsio/text"
+)
+
+type Bytes struct {
+	*zsio.Writer
+	buf bytes.Buffer
+}
+
+func (b *Bytes) Bytes() []byte {
+	return b.buf.Bytes()
+}
+
+func NewBytes(format string, tc *text.Config) (*Bytes, error) {
+	// On close, zsio.Writer.Close(), the zson WriteFlusher will be flushed
+	// then the bufwriter will closed (which will flush it's internal buffer
```

Comment:
will closed -> will be closed (?)

Reply:
stale comment doesn't apply here.  removed.